### PR TITLE
Log targets updates.

### PR DESCRIPTION
**Pull request by iamAzeem**
Log targets updates:
- ConfigMap and volume mount for the configuration.
- Inclusion of `stdout` as a log target.
- Testing of ElasticSearch.
- Testing of S3.

### CODE REVIEW

**Comment by OriHoch** [comment]
everything needs to be configurable from helm values, at the moment it seems that the helm chart is not configurable

**Comment by iamAzeem** [comment]
Right. I'll update the ConfigMap with their corresponding default helm values.

File: LOGGING.md [hunk 1]
```diff
@@ -115,12 +115,37 @@ And, can be deployed without `--dry-run`:
...
+
+**NOTE**: Only a small subset of the required configurations has been exposed
+via the environment variables.
+
+The logger's ConfigMap provides direct access to the fluentd configuration. The
+log targets can easily be configured for a more complex setup. For more details,
+please visit their respective pages:
+
+- ElasticSearch: https://docs.fluentd.org/output/elasticsearch
+- S3: https://docs.fluentd.org/output/s3
```
Comment (OriHoch) [inline LOGGING.md:134]: the helm chart should not allow all possible fluentd configurations to be set, it should only use specific, supported and documented configurations, all based on the helm values.yaml

Comment (iamAzeem) [reply LOGGING.md:134]: Noted. Will update accordingly.

**Review by OriHoch — APPROVED** [review]
looks good, I made some minor changes, you can see in last commits

**Pull request merged**